drivers/usb_c/tcpc/Kconfig.tcpc_tcpci

```diff
@@ -0,0 +1,36 @@
+# USB-C Parade PS8xxx TCPC device configuration options
+
+# Copyright (c) 2023 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+config USBC_TCPC_TCPCI
+	bool
+	help
+	  Enable support for Type-C Port Controller Interface
+
+if USBC_TCPC_TCPCI
+
+config USBC_TCPC_TCPCI_I2C_RETRIES
+	int "I2C communication retries"
+	default 2
+	help
+	  Number of I2C transaction tries that will be performed for each request.
+	  Some TCPCs are going into deep sleep mode when no charger is connected and won't respond
```

Comment:
formatting

Reply:
What do you mean? If you think about text being too long, the zephyr uses the 100 columns per line instead of 80